src/SkyImages.jl

```diff
@@ -449,86 +449,6 @@ world_to_pix{T <: Number}(patch::SkyPatch, world_loc::Vector{T}) =
 
 
 """
-A fast function to determine which sources might belong to which tiles.
-
-Args:
-  - tiles: A TiledImage
-  - patches: A vector of patches (e.g. for a particular band)
-
-Returns:
-  - An array (over tiles) of a vector of candidate
-    source patches.  If a patch is a candidate, it may be within the patch radius
-    of a point in the tile, though it might not.
-"""
-function local_source_candidates(
```

Comment:
We may want to get rid of `SkyImages.jl` entirely...most of what's left seems to belong in `ModelInit.jl` or one of the files that we're going to merge into Celeste from the `SloanDigitalSkySurvey` package.